api/lib/opentelemetry/trace/span.rb

```diff
@@ -77,10 +77,10 @@ def set_attribute(key, value)
       #
       # @param [optional String] name Optional name of the event. This is
       #   required if a block is not given.
-      # @param [optional Hash{String => String, Numeric, Boolean}] attributes
-      #   One or more key:value pairs, where the keys must be strings and the
-      #   values may be string, boolean or numeric type. This argument should
-      #   only be used when passing in a name.
+      # @param [optional Hash{String => String, Numeric, Boolean, Array<String, Numeric, Boolean>}]
+      #   attributes One or more key:value pairs, where the keys must be
+      #   strings and the values may be string, boolean or numeric type. This
```

Comment:
Comment should be updated to reflect that `values may be (array of) string, boolean or numeric type.`